tap/stats_tracker.go

```diff
@@ -23,6 +24,7 @@ type StatsTracker struct {
 	reassembledTcpPayloadsCountMutex sync.Mutex
```

Comment:
@gadotroee why are there multiple `sync.Mutex`es here instead of just one in `AppStats` struct?

Reply:
For increments/decrements using https://pkg.go.dev/sync/atomic might yield better performance.

Reply:
yes we already taught about replacing it with atomic, the code will be much better and as you said might bring better performance. you can see that one of the counters that was added lately used this atomic (I think) - but if you want to add it - your are more than welcome to do it

Reply:
@gadotroee let's do that in a later PR. Because it's not the concern of this PR. I just wanted to note that.